server/backends/blobstore/blobstore.go

```diff
@@ -255,8 +255,15 @@ func NewAwsS3BlobStore(awsConfig *config.AwsS3Config) (*AwsS3BlobStore, error) {
 		uploader:   s3manager.NewUploader(sess),
 	}
 
-	if err := awsBlobStore.createBucketIfNotExists(ctx, awsConfig.Bucket); err != nil {
-		return nil, err
+	// S3 access points can't modify or delete buckets
+	// https://github.com/awsdocs/amazon-s3-developer-guide/blob/master/doc_source/access-points.md
+	const S3_ACCESS_POINT_PREFIX = "arn:aws:s3"
```

Comment:
nit: typically constants in go are named like normal variables. Can we call this s3AccessPointPrefix instead?

Reply:
fixed!